transcendence-app/src/user/api42.service.ts

```diff
@@ -42,6 +42,7 @@ export class Api42Service {
   async get42UserData(accessToken: string): Promise<UserDto> {
     const data = await this.fetch42UserData(accessToken);
     const user: UserDto = {
+      id: undefined,
```

Comment:
As said in a comment bellow, If we all agree in getting the id from the db, I'll just remove this

Reply:
Well, I would say that the whole user.service, with it's internal UserDTO[], is a mock that we are using as a temporary workaround standing in for the missing DB connection. This means that as you implement the internals of the DB connection you are expected to completely change and remove many parts of this.